lib/hosts.tpl

```diff
@@ -32,6 +32,13 @@ hv01 glance_public_endpoint=True
 [cinder]
 hv02 cinder_ip=10.0.0.11 pvs=["/dev/sdb","/dev/sdc","/dev/sdd","/dev/sde"]
 
+## network configuration for bond0 (implemented by network-hook if defined)
```

Comment:
Please make the example more generic like the above ones. Also, our example should be with two interfaces as that's more likely to be what we see in the wild.